examples/flight/app/js/ui/todo_list.js

```diff
@@ -34,8 +35,11 @@ define([
 
 		this.edit = function (e, data) {
 			var $todoEl = $(data.el).parents('li');
+			var $inputEl = $todoEl.find('input');
+			var value = $todoEl.find('label').html();
```

Comment:
Just wondering, is this a safe thing to do? Could this have implications for HTML that's entered?

Reply:
@passy Should I trim the " $todoEl.find('label').html(); " ?

Reply:
`.html()` => `.text()`